Change information about using CNAME records pointing to dns.umbraco.io (#6080)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/manage-hostnames/README.md

<figure><img src="../../../.gitbook/assets/image (26).png" alt="Manage hostnames"><figcaption><p>Manage hostnames</p></figcaption></figure>

> Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service. The DNS settings can either use a CNAME or an A & AAAA records:

Reviewer: ```suggestion
Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service. The DNS settings can either use a CNAME or an A & AAAA record:
```

> - A & AAAA records: they are usually used for the Apex domain (without "`www`" in the URL), and needs to be created at the root of your domain.
> 
> - CNAME: usually used for domains with "`www`" in the URL, recommended to use if possible, as the record is not changed as often as A & AAAA records are. When setting up a CNAME it needs to point to `dns.umbraco.io`.

Reviewer: ```suggestion
- **A & AAAA** records: they are usually used for the Apex domain (without "`www`" in the URL), and needs to be created at the root of your domain.

- **CNAME**: usually used for domains with "`www`" in the URL, recommended to use if possible, as the record is not changed as often as A & AAAA records are. When setting up a CNAME it needs to point to `dns.umbraco.io`.
```

> {% hint style="info" %} Be aware that when using dns.umbraco.io, wildcard asterisks are not supported. Individual CNAME records must be created for each individual subdomain you want to use `dns.umbraco.io`. {% endhint %}

Reviewer: ```suggestion
{% hint style="info" %} Be aware that when using dns.umbraco.io, wildcard asterisks are not supported. Individual CNAME records must be created for each subdomain you want to use `dns.umbraco.io`. {% endhint %}
```

> Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service.
> 

Reviewer: ```suggestion
Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service. The DNS settings can either use a CNAME or an A & AAAA records:

```

> A & AAAA records should be created for the root of your domain.
> 
> For any subdomain, we recommended setting a CNAME to either the root of your domain or to dns.umbraco.io.

Reviewer: I have updated this part a bit to make it more clear when to use what. I have also removed the part where it says to add cname at the root to not confuse others and as in current docs we recommend to point to dns.umbraco.io and not another domain that points to this. When this is confirmed internally the docs will be updated as well. I have also removed mention of subdomain from CNAME as it is already in the info note. 
```suggestion
- A & AAAA records: they are usually used for the Apex domain (without "`www`" in the URL), and needs to be created at the root of your domain.

- CNAME: usually used for domains with "`www`" in the URL, recommended to use if possible, as the record is not changed as often as A & AAAA records are. When setting up a CNAME it needs to point to `dns.umbraco.io`.
```

> {% hint style="info" %} Be aware that when using dns.umbraco.io, wildcard asterisks are not supported. Individual CNAME records must be created for each individual subdomain you want to use dns.umbraco.io. {% endhint %}

Reviewer: ```suggestion
{% hint style="info" %} Be aware that when using dns.umbraco.io, wildcard asterisks are not supported. Individual CNAME records must be created for each individual subdomain you want to use `dns.umbraco.io`. {% endhint %}
```
